okteto.yml

```
@@ -1,17 +1,23 @@
-name: supervisor
-image: okteto/golang:1
-command:
-- bash
-workdir: /okteto
-volumes:
-- /go/pkg/
-- /root/.cache/go-build/
-securityContext:
-  capabilities:
-    add:
-    - SYS_PTRACE
-forward:
-- 8080:8080
-- 2345:2345
-persistentVolume:
-  enabled: true
\ No newline at end of file
+build:
+  supervisor:
+    context: .
+    image: ${REGISTRY_AND_USER:-okteto}/supervisor:${VERSION:-latest}
```

what do you think if we use some more conventional env vars?

If we use `OKTETO_NAMESPACE` instead of USER it's replaced out of the box. For the Registry, unfortunatelty it's only injected in the deploy steps, but still we could have a comment as example.

```suggestion
    image: ${OKTETO_REGISTRY_URL}/${OKTETO_NAMESPACE}/supervisor:${VERSION:-latest}
    # Example: OKTETO_REGISTRY_URL=registry.product.okteto.dev okteto build`
```

---

With that approach it's difficult to know that you can push to other registries rather than the okteto one, like docker hub.